packages/quantic/scripts/npm/setup-quantic.js

```diff
@@ -0,0 +1,123 @@
+const fs = require('fs');
+const pathlib = require('path');
+
+function copyFileSync(source, target) {
+  try {
+    let targetFile = target;
+    if (fs.existsSync(target) && fs.lstatSync(target).isDirectory()) {
+      targetFile = pathlib.join(target, pathlib.basename(source));
+    }
+
+    fs.writeFileSync(targetFile, fs.readFileSync(source));
+  } catch (err) {
+    console.error(`Failed to copy file at the following path ${source}`);
+    console.error(err);
+    process.exit(1);
+  }
+}
+
+function copyFolderRecursiveSync(source, target) {
```

Comment:
PostInstall has access to npm dependencies. So save yourself some trouble and find a dependency that does that for you, fs-extra prolly would do the trick

Reply:
I agree that using a dependency here would make the script much simpler 👌